core/sqf/sql/scripts/sqstart

```diff
@@ -245,6 +245,39 @@ function checkKerberos {
    return 0
 }
 
+function checkUlimit {
+  
+ #check max_pid 
+ userProc=`cat /proc/sys/kernel/pid_max`
+ if [[ $userProc -gt 65535 ]] ; then
+    echoLog ""
+    echoLog "ERROR: detect issue during startup, please make sure your kernel parameter (sysctl) setup correctly"
+    echoLog "ERROR: max_pid should not be greater than 65535"
+    echoLog ""
+    return
+ fi
+
+ #check ulimit -l
+ maxLockMem=`ulimit -l`
+ if [[ $maxLockMem -lt 65536 ]] ; then
+    echoLog ""
+    echoLog "ERROR: detect issue during startup, please make sure your ulimit setup correctly"
```

Comment:
Suggested wordsmith: "ERROR: issue detected during startup. Please make sure your ulimit is set up correctly."